components/post_view/post_list.jsx

```diff
@@ -115,6 +115,7 @@ export default class PostList extends React.PureComponent {
         this.postListContentRef = React.createRef();
         this.scrollAnimationFrame = null;
         this.resizeAnimationFrame = null;
+        this.atBottom = false;
```

Comment:
Why are these changes still needed? Is the snapshot not enough here to keep you at the bottom?

Reply:
yes. These are needed because snaphot lifecycle will component updates but here things like images need to trigger scroll correction which happens outside of lifecycle so when a image loads say at bottom now we need to tell to manually scroll to bottom based on previous flag i,e `atBottom` flag here

Reply:
Let me give this another thought.